docs/reference/mapping/types.asciidoc

```diff
@@ -69,6 +69,7 @@ values.
 ==== Text search types
 
 <<text,`text`>>:: Analyzed, unstructured text.
+<<match-only-text,`match_only_text`>>:: A more space-efficient variant of `text`.
```

Comment:
We decided to make a single page for the `keyword` type family, so that users could easily compare the trade-offs between each type. between It'd be nice to do the same for the new 'text' family'.